src/lib/tls/tls_callbacks.h

```diff
@@ -125,19 +122,20 @@ class BOTAN_PUBLIC_API(2,0) Callbacks
          }
 
        /**
-       * Optional callback: New session ticket received
-       * Called when we receive a session ticket from the server at any point
-       * after the initial handshake has finished. Clients may decide to keep or
-       * discard the session ticket in the configured session manager.
+       * Optional callback: Resumption information was received/established
+       *
+       * TLS 1.3 calls this when we sent or received a TLS 1.3 session ticket at
```

Comment:
Maybe make it more clear that (on the server side) for sending, this is called *before* the session is sent (or even the session manager asked for one).

So it will also prevent sending of the information no just the caching.